test/e2e/set/scyllacluster/scyllamanager.go

```diff
@@ -266,83 +272,459 @@ var _ = g.Describe("Scylla Manager integration", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(tasks.TaskListItemSlice).To(o.BeEmpty())
 	})
+})
+
+var _ = g.Describe("Scylla Manager integration (requiring object storage)", framework.RequiresObjectStorage, func() {
```

Comment:
```suggestion
var _ = g.Describe("Scylla Manager integration", framework.RequiresObjectStorage, func() {
```
it's already there from the label
`Scylla Manager integration (requiring object storage) should register cluster, sync backup tasks and support manual restore procedure [RequiresObjectStorage]`

Reply:
done